targets/TARGET_STM/pwmout_api.c

```diff
@@ -214,10 +251,13 @@ void pwmout_write(pwmout_t *obj, float value)
             return;
     }
 
-    if (HAL_TIM_PWM_ConfigChannel(&TimHandle, &sConfig, channel) != HAL_OK) {
+    if (LL_TIM_CC_IsEnabledChannel(TimHandle.Instance, TIM_ChannelConvert_HAL2LL(channel, obj)) == 0) {
```

Comment:
Maybe add a small comment here to explain the 2 cases, basically:
- if not yet active, a full initialization will be done 
- if channel is already active, only the comparator will be set, which avoids possible glitches

Reply:
I checked this is fixed ok now

Reply:
Done